dokka-subprojects/analysis-kotlin-symbols/src/main/kotlin/org/jetbrains/dokka/analysis/kotlin/symbols/plugin/KotlinAnalysis.kt

```diff
@@ -62,6 +63,9 @@ internal fun getLanguageVersionSettings(
             // special flag for Dokka
             // force to resolve light classes (lazily by default)
             AnalysisFlags.eagerResolveOfLightClasses to true
+        ),
+        specificFeatures = mapOf(
+            LanguageFeature.MultiPlatformProjects to LanguageFeature.State.ENABLED,
```

Comment:
Should we enable it only if we have a truly multi-platform project? For example, in case we have at least one source set with a non-JVM platform?

It would be nice to ask the AA/FE teams regarding the consequences of enabling this flag.